src/main/java/com/actiontech/dble/route/RouteResultsetNode.java

```diff
@@ -124,6 +137,9 @@ public void setLoadData(LoadData loadData) {
 
     @Override
     public int hashCode() {
+        if (LoadDataBatch.getInstance().isEnableBatchLoadData() && !Objects.isNull(loadData) && !Strings.isNullOrEmpty(loadData.getFileName())) {
```

Comment:
maybe cause problems in some hashmap.
for example,
 two object A and B,
which A equals B, but A.hashCode() not equals B.hashCode().